check/base.lua

```diff
@@ -427,14 +427,14 @@ function ChildCheck:_runChild(exePath, exeArgs, environ, callback)
 
   self._log(logging.DEBUG, fmt("%s: starting process", exePath))
 
-  local child = childprocess.spawn(exePath, exeArgs, { env = environ })
+  local child = childprocess.spawn(exePath, exeArgs, { env = environ, detached = true})
```

Comment:
Why change checks to spawn detached? This seems like it introduces a huge bug where when the agent is terminated, check processes will continue running, maybe even when the agent is shut down normally.